Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -3580,6 +3580,10 @@ WI.reset = async function()
 
 WI.isEngineeringBuild = false;
 
+WI.engineeringSettingsAllowed = function() {
+    return WI.isEngineeringBuild || InspectorFrontendHost.engineeringSettingsAllowed();
```

Comment:
Do we even need `WI.isEngineeringBuild` anymore?  Or is this just a convenience to allow engineers to skip changing that preference for local builds?

Also, should we maybe cache the result of `InspectorFrontendHost.engineeringSettingsAllowed()` somewhere?

Reply:
Depending on a platform's decision in `InspectorFrontendHost.engineeringSettingsAllows()`, `WI.isEngineeringBuild` can still be useful for open-source contributors to access the functionality.

Caching the result of `InspectorFrontendHost.engineeringSettingsAllows()` makes sense.